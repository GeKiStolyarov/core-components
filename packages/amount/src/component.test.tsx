import React from 'react';
import { render } from '@testing-library/react';
import { Amount } from './index';

describe('Amount', () => {
    it('should match snapshot', () => {
        const testId = 'test-id';

        const { getByTestId } = render(
            <Amount value={100} currency='RUR' minority={100} dataTestId={testId} />,
        );
        const amount = getByTestId(testId);
        expect(amount).toMatchSnapshot();
    });
});
