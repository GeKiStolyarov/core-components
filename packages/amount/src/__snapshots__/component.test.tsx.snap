// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Amount should match snapshot 1`] = `
<div
  class="component amount"
  data-test-id="test-id"
>
  1
  <span
    class="minorPart"
  >
    ,
    00
     
    ₽
  </span>
</div>
`;
